docs/admin/auth/hba.rst

```diff
@@ -62,12 +62,19 @@ the client given that this user exists.
 If the setting ``auth.host_based`` is present and the configurations list does
 not contain any entry, then no client can authenticate.
 
+Default JWT configuration is defined in the ``auth.host_based.jwt`` setting in
+the ``crate.yml`` file. Allowed fields are ``iss`` and ``aud``. The description
+of these fields can be found in :ref:`jwt_defaults`.
```

Comment:
Shouldn't we also mention here that, if the global JWT config is enabled, all user-defined properties are ignored? I think this is quite important to know **before** enabling these settings.

Reply:
it's mentioned in the referenced `:ref:jwt_defaults`.

https://github.com/crate/crate/pull/16763/files#diff-cb98a5cbb83b46f8a3764e3698973d91b2bbef720ff0a8577aaa14e56d688075R567

Added same note here for better visibility of this message